src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -2923,8 +2923,12 @@ public double getHumidity(int x, int y, int z)
 	@Override
 	public int getLogicalHeight()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return switch (environment)
+		{
+			case NETHER, THE_END -> 256;
+            case NORMAL -> 384;
```

Comment:
Code style

Reply:
Whats wrong here?

Reply:
Spaces not tabs. I was on mobile so it looked like the indentation was wrong, but it should be corrected either way.